bigquery/metadata.py

```diff
@@ -0,0 +1,42 @@
+import os
+from typing import TypedDict, Tuple, Dict, List
+
+from google.cloud import bigquery
+
+
+class Segment(TypedDict):
+    id: int
+    city: str
+    start_postmile: float
+    end_postmile: float
+    representative_point: Tuple[float, float]
```

Comment:
Do we have the coordinates of the start point and end point?

Reply:
Yes, they're already in `metadata.json` and I've just added them to this type hint.